msteams-platform/apps-in-teams-meetings/API-references.md

```diff
@@ -413,6 +413,33 @@ The JSON response body for Meeting Details API is as follows:
     } 
     ```
 
+* **Scheduled channel meetings:**
+
+    ```json
+    { 
+        "details": { 
+        "msGraphResourceId": "MSoxNmUwYjdiYi05M2Q1LTQzNTItOTllMC0yM2VlNWYyZmZmZTIqMTY2MDc1ODYwNzc0MCoqMTk6a0RtQkpEWFZsYWl0QWhHcVB2SzBtRExZbHVTWnJub01WX1MxeFNkTjQxNDFAdGhyZWFkLnRhY3Yy", 
+        "scheduledStartTime": "2022-08-17T18:00:00Z", 
+        "scheduledEndTime": "2022-08-17T18:30:00Z", 
+        "type": "ChannelScheduled", 
```

Comment:
pls add the same in property description in below table

Reply:
Updated as mentioned.